notebooks/20220512_explore_16s.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>Did you mean to have this in the notebook? It doesn't actually pull up the package info within the notebook so I'm not sure how helpful it is.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/2022-cf-sputum/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>nope! will remove, thank you!</p>